ci/tasks/test-gcs-blobstore-client-integration.yml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Where will this task be run? Are you planning to add it to the main [pipeline](https://github.com/Everlag/bosh/blob/master/ci/pipeline.yml)?